CHANGELOG.md

```diff
@@ -1,2 +1,10 @@
+## 2.0.1
```

Comment:
We do not have releases of 2.x yet, so no changelog is neccessary

Reply:
removed